lib/local_system.rb

```diff
@@ -49,8 +49,10 @@ def validate_machinery_compatibility
       supported_oses = Os.supported_host_systems.map { |o| o.canonical_name }.sort.join(", ")
       message = <<EOF
 You are running Machinery on a platform we do not explicitly support and test.
-It still could work very well. If you run into issues or would like to provide us feedback, you are welcome to file an issue at https://github.com/SUSE/machinery/issues/new or write an email to machinery@lists.suse.com.
```

Comment:
I wouldn't remove the period at the end but instead just add a space between the mail address and the period.